api/envoy/config/metrics/v2/stats.proto

```diff
@@ -53,8 +53,14 @@ message StatsConfig {
   // <https://github.com/envoyproxy/envoy/blob/master/source/common/config/well_known_names.h>`_
   // for a list of the default tags in Envoy.
   //
-  // If not provided, the value is assumed to be true.
+  // If not provided, this value defaults to true.
   google.protobuf.BoolValue use_all_default_tags = 2;
+
+  // Flush each individual statistic to the configured sinks. If this is set to false, each sink
+  // will be responsible for querying the stats manually in startFlush() and endFlush().
+  //
+  // If not provided, this value defaults to true.
+  google.protobuf.BoolValue flush_individual_stats = 3;
```

Comment:
Can we infer from the configured StatsSink whether this should be true or false? Or is it really useful to make this manually configurable?

If we leave this manually configurable, can you please document more specifically when this should be set to false?

Also, I think we're trying to use "bool" instead of "google.protobuf.BoolValue" for new values. Can we invert the name so that "false" is the current behavior (the default)?

Reply:
Sounds good, I'll wait for others to weigh in on the former.  As for the latter, that totally makes sense.  I'll move to a simple bool.

Reply:
@envoyproxy/maintainers Would like opinions on this.